config/locales/errors/fr.yml

```diff
@@ -26,27 +28,33 @@ fr:
         un autre.
       expired: est expiré, veuillez en demander un nouveau
       format_mismatch: Veuillez vous assurer de respecter le format requis.
-      improbable_phone: NOT TRANSLATED YET
-      invalid_calling_area: NOT TRANSLATED YET
-      invalid_phone_number: NOT TRANSLATED YET
-      invalid_sms_number: NOT TRANSLATED YET
+      improbable_phone: Numéro de téléphone non valide. Veillez à saisir un numéro
+        de téléphone valide.
+      invalid_calling_area: Les appels vers ce numéro de téléphone ne sont pas pris
+        en charge. Veuillez essayer par SMS si vous possédez un téléphone disposant
+        de cette fonction.
+      invalid_phone_number: Le numéro de téléphone saisi n'est pas valide.
+      invalid_sms_number: Le numéro de téléphone saisi ne prend pas en charge les
+        messages textuels. Veuillez essayer l'option d'appel téléphonique.
       invalid_voice_number: Numéro de téléphone invalide. Vérifiez que vous avez entré
         le bon indicatif international ou régional.
       missing_field: Veuillez remplir ce champ.
-      must_have_us_country_code: NOT TRANSLATED YET
+      must_have_us_country_code: Numéro de téléphone non valide. Veuillez vous assurer
+        de saisir un numéro de téléphone avec un code pays des États-Unis.
       no_pending_profile: Aucun profil en attente de vérification
-      otp_failed: NOT TRANSLATED YET
+      otp_failed: Échec de l'envoi de votre mot de passe à usage unique.
```

Comment:
Heh. This is the correct translation of the English text we have, but the English text is wrong. In English, it should say "your security code failed to send", not "your one time password failed to send'. In French, it would be `Échec de l'envoi de votre code de sécurité`.

We should probably also update the voice.otp.message` text to refer to security code or confirmation code to be consistent with SMS. Currently, we still use "one time passcode" for voice. That's the only place in the app where we use the term "passcode".